netsim/augment/links.py

```diff
@@ -298,8 +298,8 @@ def add_node_interface(node: Box, ifdata: Box, defaults: Box) -> Box:
       if sys_mtu and node.mtu == ifdata.mtu:    # .. is it equal to node MTU?
         ifdata.pop('mtu',None)                  # .... remove interface MTU on devices that support system MTU
     else:                                       # Node MTU is defined, interface MTU is not
-      if not sys_mtu:                           # .. does the device support system MTU?
-        ifdata.mtu = node.mtu                   # .... no, copy node MTU to interface MTU
+      if not sys_mtu and ifdata.get('type',None)!='lag':  # .. does the device support system MTU?
```

Comment:
And why would we want to ignore system MTU on LAG interfaces?

Reply:
On Linux the bond mtu is derived from the member ports, it won't accept setting mtu on the bond device. But that condition should be applied there, not here